.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: javascript
 
+before_script:
+    - npm install angular karma karma-jasmine jasmine-core angular-mocks karma-phantomjs-launcher
```

Comment:
Is this even necessary if you already put it into packages.json? I think we should call simply npm install